include/ck/tensor_operation/gpu/element/element_wise_operation.hpp

```diff
@@ -144,6 +144,21 @@ struct AddHardswishAdd
     }
 };
 
+struct Relu
+{
+    __host__ __device__ void operator()(float& y, const float& x) const { y = x > 0 ? x : 0; }
+
+    __host__ __device__ void operator()(half_t& y, const half_t& x) const { y = x > 0 ? x : 0; }
+
+    __host__ __device__ void operator()(bhalf_t& y, const bhalf_t& x) const { y = x > 0 ? x : 0; }
```

Comment:
I think we could not using bhalf_t  for comparing directly,  we should do either
1)  In the operator() overloaded for `bhalf_t`,  convert the input to float temporarily before comparing                      or
2)  Does not overload the operator() for `bhalf_t`,  force the application to use `float` as `ComputeDataType` when `bhalf_t` is involved

Reply:
I fix 1 in 3e4d27529ee2ebf3b3a42f468f07129cfe016a78